packages/fuel-indexer-benchmarks/src/bin/qa.rs

```diff
@@ -505,6 +518,45 @@ fn record_mem_usage() -> Result<String, std::io::Error> {
         .read()
 }
 
+fn record_active_connections() -> Result<usize, std::io::Error> {
+    let proc = Command::new("psql")
+        .arg("-U")
+        .arg("postgres")
+        .arg("-c")
+        .arg("SELECT count(*) used FROM pg_stat_activity")
+        .arg("--no-align")
+        .arg("--tuples-only")
+        .stdout(Stdio::piped())
+        .stderr(Stdio::piped())
+        .spawn()
+        .unwrap();
+    let output = proc.wait_with_output().unwrap();
+    let output = String::from_utf8(output.stdout).unwrap();
+    let output = output.trim();
+    let output = output.parse::<usize>().unwrap_or(0);
+    println!("Active connections: {output}");
```

Comment:
We can either set this behind `debug` log level or remove it @lostman - currently running the script right now, this is a bit too much spam